resources/wf_builder.rb

```diff
@@ -96,24 +96,23 @@
     chef_file "#{workspace}/#{dir}/builder_key" do
       source new_resource.builder_pem
       mode '0600'
-      user 'root'
-      group 'root'
+      user 'dbuild'
+      group 'dbuild'
     end
 
     chef_file "#{workspace}/#{dir}/#{chef_user}.pem" do
       source new_resource.chef_user_pem
       mode '0600'
-      user 'root'
-      group 'root'
+      user 'dbuild'
+      group 'dbuild'
     end
   end
 
   %w(etc/delivery.rb .chef/knife.rb).each do |dir|
     file "#{workspace}/#{dir}" do
       content ensurekv(::File.read('/etc/chef/client.rb'),
                        node_name: new_resource.chef_user,
-                       log_location: 'STDOUT',
```

Comment:
I'll add that back in and see if it works, I think the permission errors were related to the keys.